app/controllers/comments_controller.rb

```diff
@@ -1,8 +1,8 @@
 class CommentsController < ApplicationController
-  before_action :find_article, only: [:create, :destroy]
   def create
+    @article = Article.find(params[:article_id])
```

Comment:
create a `before_action` filter with name of `find_article` and put that line in that method